app_invoiceninja.md

```diff
@@ -0,0 +1,24 @@
+# Invoice Ninja
+
+[![Install Invoice Ninja with YunoHost](https://install-app.yunohost.org/install-with-yunohost.png)](https://install-app.yunohost.org/?app=invoiceninja)
+[![Integration level](https://dash.yunohost.org/integration/invoiceninja.svg)](https://dash.yunohost.org/appci/app/invoiceninja) ![](https://ci-apps.yunohost.org/ci/badges/invoiceninja.status.svg) ![](https://ci-apps.yunohost.org/ci/badges/invoiceninja.maintain.svg)  
```

Comment:
```suggestion
[![Integration level](https://dash.yunohost.org/integration/invoiceninja.svg)](https://dash.yunohost.org/appci/app/invoiceninja)
```
I think only integration badge is used in these app_doc pages.

Reply:
Can you commit this? Since this PR was created "anonymously" I cannot change it.